disnake/embeds.py

```diff
@@ -268,17 +268,20 @@ def copy(self) -> Self:
         embed._files = self._files.copy()
         return embed
 
-    def __len__(self) -> int:
-        total = len(self.title or "") + len(self.description or "")
+    def __len__(self):
+        def stripped_length(text):
+            return len(text.strip())
```

Comment:
This is a nice utility method but calling it incurs a fairly heavy performance hit ([benchmark](https://paste.disnake.dev/?id=1658876451296794)).
Even if it might look slightly worse, it would be better to do these calls inline (e.g. `len(field["name"].strip())` instead of `stripped_length(field["name"])`).